src/main/java/com/heatheryou/httpserver/CommandLineArgs.java

```diff
@@ -0,0 +1,23 @@
+package com.heatheryou.httpserver;
+
+public class CommandLineArgs {
+    public static void displayErr(ISystemOutput systemOutput) {
+        systemOutput.printErr("Usage: java -jar http-server.jar <port number>");
+        systemOutput.exit(1);
+    }
```

Comment:
This almost looks like it's doing two things?  The `isInvalid` and `parsePort` seem to belong here, but the `displayErr` almost looks like it belongs somewhere else?